scripts/compute_average_distance.py

```
@@ -0,0 +1,547 @@
+
+
+def main() -> int:
+    ap = argparse.ArgumentParser(description=__doc__)
+    g = ap.add_mutually_exclusive_group()
+    g.add_argument("--validate", action="store_true", help="既知路線で較正のみ（CSV不変）")
+    g.add_argument("--apply", action="store_true", help="全路線を計算しCSVを書き換える")
+    g.add_argument("--lines", type=str, help="カンマ区切りの line_cd を計算（CSV不変）")
+    args = ap.parse_args()
+
+    lines = load_lines()
+    lines_meta = {l["line_cd"]: l for l in lines}
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)